Sources/PointFree/StripeWebhooks.swift

```diff
@@ -200,3 +204,16 @@ private func stripeHookFailure<A>(
       }
     }
 }
+
+private func extraSubscriptionId(
+  fromEvent event: Event<Either<Invoice, Stripe.Subscription>>
+  ) -> Stripe.Subscription.Id? {
+
+  switch event.data.object {
+  case let .left(invoice):
+    return invoice.subscription
+      ?? invoice.lines.data.compactMap(^\.subscription).first
+  case let .right(subscription):
+    return subscription.id
```

Comment:
writing this function makes me realizing that we could maybe process hooks with something parser-like to maybe clean them up a bit.